src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -25,7 +28,10 @@ public ProjectFileReader(INugetRestoreProcess nugetRestoreProcess = null)
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<ProjectFileReader>();
         }
 
-        public IAnalyzerResult AnalyzeProject(string projectFilePath, string solutionFilePath)
+        public IAnalyzerResult AnalyzeProject(
+            string projectFilePath,
+            string solutionFilePath,
+            string targetFramework)
```

Comment:
Why does the targetFramework not have a default null value? It has on the interface. If we add that the unit tests also could use the default